marlowe-playground-client/src/Demos/Types.purs

```diff
@@ -0,0 +1,21 @@
+module Demos.Types where
+
+import Prelude
+import Analytics (class IsEvent, Event)
+import Data.Maybe (Maybe(..))
+import Data.Newtype (class Newtype)
+import Projects.Types (Lang)
+
+newtype Demo
+  = Demo String
+
+derive instance newtypeDemo :: Newtype Demo _
+
+data Action
+  = LoadDemo Lang Demo
+
+defaultEvent :: String -> Event
+defaultEvent action = { category: Just "Demos", action, label: Nothing, value: Nothing }
+
+instance isEventAction :: IsEvent Action where
+  toEvent (LoadDemo lang _) = Just { category: Just "Demos", action: "LoadDemo", label: Just (show lang), value: Nothing }
```

Comment:
Better, but `defaultEvent` is still either redundant or underused, right?

Reply:
oh, I thought I'd deleted it, will do